x/docs/md/cli/windows.md

```diff
@@ -1,80 +1,110 @@
+Chocolately lets you install Windows applications quickly and easily. To install Chocolatey please refer to the [installation instructions on the Chocolatey website](https://chocolatey.org/install). 
 
-```powershell
-function which($name) { Get-Command $name -ErrorAction SilentlyContinue | Select-Object Definition }
+> **NOTE**
+> Please ensure that when you open up your Windows shell, whether it's the Command Prompt or Powershell, you open it with **Administrator privileges**, otherwise the Chocolatey installation will fail. If you are unsure how to do this, please refer to the [help below](#open-win-cmd).
 
-if ((which cinst) -eq $null) {
-    iex ((new-object net.webclient).DownloadString('https://chocolatey.org/install.ps1'))
-    $env:Path = [System.Environment]::GetEnvironmentVariable("Path","Machine") + ";" + [System.Environment]::GetEnvironmentVariable("Path","User")
-}
+If you want to check if your Chocolatey installation has been successful, please refer to the [help below](#test-choco-install).
+
+### Installing the Exercism CLI <a name="install-exercism-cli"></a>
+
+With Chocolatey installed, you can now install the Exercism CLI. In your Windows shell, either the Command Prompt or Powershell, type in the following:
 
-cinst exercism-io-cli
 ```
+choco install exercism-io-cli
+```
+> **NOTE**
+>  You don't need to open your Windows shell with Administrator privileges to use the `choco` command.
 
-#### Using the Command Prompt
-1. Open a command line interface (CLI) by clicking on "Start", typing "cmd" into the search bar and pressing enter
-1. Next copy and paste the following command into the command window:
+Type 'Y' to accept any prompts during the installation. If the installation was successful you should have output similar to the following:
 
-        @powershell -NoProfile -ExecutionPolicy unrestricted -Command "iex ((new-object net.webclient).DownloadString('https://chocolatey.org/install.ps1'))" && SET PATH=%PATH%;%ALLUSERSPROFILE%\chocolatey\bin
+![Exercism CLI installation](../../../../public/img/cli/win-exercism-installation.png "Exercism CLI installation")
 
-1. Once the installation finishes, Chocolatey should be installed. To test the installation you can open a new command window (step 1) and enter the following command:
+You now have access to the `exercism` command. You can type in `exercism --version` to get the current version of the Exercism CLI, or `exercism --help` to show the help menu.
 
-        choco /?
+![Exercism command](../../../../public/img/cli/win-exercism-command.png "Exercism command")
```

Comment:
That would be _awesome_. One of the hardest parts of writing the documentation for this is that we have no idea what we assume to be obvious and forget to put in the documentation!